docs/workshops/intro-to-accessibility.md

```diff
@@ -72,8 +69,392 @@ Try using your phone with the screen reader function.
+
+#### Vision impairments
+
+In the UK: almost 2 million people living with sight loss.
+360,000 are registered as blind or partially sighted.
+
+**Blindness**: there is a legal definition which doesn’t imply seeing nothing at all. Many can read from a screen, but can’t use a mouse.
+
+They often can’t see images, graphics or videos.
+
+If the layout of your page, the navigation or labels of your fields are not clear, they won’t be able to use your website or app.
+
+The main screen readers are:
+
+- NVDA - it’s free
+- JAWS - need a licence but good and have been here for a long time
+- Voice Over
+
+2 minutes video of a user (Kristy Viers) using an iPhone
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/l0xBsR5mIYs" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+#### People with low vision
+
+- Might need to magnify and miss some information if you don’t **align your content on the left** or don’t group related items together.
+- Need to be able to scroll both vertically and horizontally
+- Can get lost in your page
+- Might need to change colour contrast on their device or browser
+
+![poster showing do and dont for screen readers](assets/a11y-part1/visually-impaired-screenreader.png)
+![poster showing do and don't for low vision](assets/a11y-part1/visually-impaired-low-vision.png)
+[Posters part of a set available here](https://github.com/UKHomeOffice/posters/blob/master/accessibility/dos-donts/posters_en-UK/accessibility-posters-set.pdf)
+
+#### Colour blindness
+
+People with colour vision deficiency find it difficult to identify and distinguish between certain colours.
+
+Total colour blindness (an inability to see any colour) is very rare.
+
+There are various types:
+
+- Deuteranomaly: reduce sensitivity to green
+- Protanopia: reduce sensitivity to red
+- Tritanopia: reduce sensitivity to blue
+
+For this people, information via colour alone won’t be understood or noticed.
+
+For example, an error message in red with no other marks that something is wrong.
+
+![4 images of tomatoes, normal vision, protanopia, tritanopia and deuteranomaly](assets/a11y-part1/different-types-color-blindness-photos-76.jpg)
+
+> **Exercise 1**:
+> Looking at this page: [Morrisons](https://groceries.morrisons.com/webshop/startWebshop.do)
+> What do you think could be problematic for:
+>
+> - a person with low vision magnifying their screen by 4 (ctlr + to 400%)
+> - a colour blind user
+>
+> Look at the pictures, the contrast, the colours
+>
+> Use Dev tool to emulate colour blindness:
+> Go to console, then 3 dots menu, then rendering and then emulation at the bottom to see any page as a colour blind person would.
+
+#### Hearing impairments
+
+About **9 million people in the UK who are Deaf or hard of hearing**
+
+- any audio without subtitles or a transcript can be problematic
```

Comment:
Capitalised at the start of the bullet points :)

Reply:
ahahaha this is an regular controversy, I'm following the GOVUK styles convention, https://design-system.service.gov.uk/styles/typography/#bulleted-lists

but if the syllabus has a different convention, I'll use that instead